lib/rules/max-statements-per-line.js

```diff
@@ -35,11 +35,11 @@ module.exports = {
         const sourceCode = context.getSourceCode(),
             options = context.options[0] || {},
             maxStatementsPerLine = typeof options.max !== "undefined" ? options.max : 1,
-            message = "This line has too many statements. Maximum allowed is {{maxStatementsPerLine}}.",
-            data = { maxStatementsPerLine };
+            message = "This line has {{numberOfStatementsOnThisLine}} statement(s). Maximum allowed is {{maxStatementsPerLine}}.";
```

Comment:
Sure, I can do that. My thinking was that having the non-variable part be identical would make it easier for people doing log parsing. Should I make this change?